www/js/homematic-manager.js

```diff
@@ -1353,13 +1353,26 @@ function initGridDevices() {
                 // {name: 'LINK_TARGET_ROLES', index: 'LINK_TARGET_ROLES', width: 100, hidden: true},
                 // {name: 'VERSION', index: 'VERSION', width: 58, fixed: true, align: 'right'}
             ],
+            multiselect: true,
             rowNum: 1000000,
             autowidth: true,
             height: 'auto',
             width: 1000,
             sortorder: 'desc',
             viewrecords: true,
             ignoreCase: true,
+            // https://stackoverflow.com/a/44781582
+            beforeSelectRow: function (rowid, e) {
+                if (e.shiftKey == true) {
+                    // SHIFT held --> multi select
+                    return true;
+                } else {
+                    // Reset existing selection
+                    $('[id^="channels_"][id$="_t"]').jqGrid('resetSelection');
+                    // Pass through new item selection
+                    return true;
+                }
+            },
```

Comment:
The only disadvantage with this implementation is that a right click on a row no longer immediately selects it. The alternative would be just having only `multiselect: true` which would mean that every click selects or deselects a row.